ggml/src/ggml-alloc.c

```diff
@@ -947,6 +948,7 @@ static bool alloc_tensor_range(struct ggml_context * ctx,
         free(*buffers);
         return false;
     }
+    ggml_backend_buffer_set_usage(buffer, GGML_BACKEND_BUFFER_USAGE_COMPUTE);
```

Comment:
The other call is good, but this function is part of `ggml_backend_alloc_ctx_tensors_from_buft`, which is used to allocate weights, KV and other static tensors, so it should not be done here.